bindings/python-examples/tests.py

```diff
@@ -53,6 +54,10 @@ def test_link_display_with_identical_link_type2(self):
         self.assertEqual(str(Link(None, 0, 'Left','Link','Link*','Right')),
                          u'Left-Link-Link*-Right')
 
+class AAADictionaryTestCase(unittest.TestCase):
+    def test_open_nonexistent_dictionary(self):
+        self.assertRaises(LG_DictionaryError, Dictionary, 'No such language')
+
```

Comment:
Since tests seem to be run in alphabetical order, it would be best to keep the code in alphabetical order too.